public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -74,43 +73,42 @@ block quickstart-repo
 .l-sub-section
   :marked
     More precisely, the redisplay occurs after some kind of asynchronous event related to
-    the view such as a keystroke, a timer completion, or an async `XHR` response.
-    We don't have those in this sample.
-    But then the properties aren't changing on their own either. For the moment we must operate on faith.
+    the view, such as a keystroke, a timer completion, or an async `XHR` response.
```

Comment:
Why is `XHR` in code font? I don't think it's a string you'll see in any code. Also, you already said "asynchronous" before, so "async" seems unnecessary. Maybe:

...a response to an HTTP request.